chart/RELEASE_NOTES.rst

```diff
@@ -23,6 +23,49 @@ Run ``helm repo update`` before upgrading the chart to the latest version.
 
 .. towncrier release notes start
 
+Airflow Helm Chart 1.17.0 (2025-06-17)
+--------------------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+Default Airflow image is updated to ``3.0.2`` (#51594)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+The default Airflow image that is used with the Chart is now ``3.0.2``, previously it was ``2.10.5``.
+
+New Features
+^^^^^^^^^^^^
+- Add extra secret annotations to most secrets (#48890)
+- Add support for EdgeExecutor (#50897)
+
+Improvements
+^^^^^^^^^^^^
+- Unify k8s labels & add some missing k8s labels (#49522)
+
+Bug Fixes
```

Comment:
https://github.com/apache/airflow/pull/51080 is missing from the changelog

Reply:
weird, wonder how I missed it...

Reply:
Oh, my local main branch is out of date :)